src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -52,6 +52,10 @@ interaction type typeSecretIn {
 	template = "Enter secret" ${value} "into" ${element}
 	method = WebDriverFixture.typeSecretInto(element, locatorStrategy, value)
 }
+interaction type typeSecretInUnsecure {
```

Comment:
I'd prefer 'insecure' over 'unsecure' (and InInsecure could become IntoInsecure ...)